samples/client/petstore/ruby-faraday/docs/FakeApi.md

```diff
@@ -457,19 +457,19 @@ Petstore.configure do |config|
 end
 
 api_instance = Petstore::FakeApi.new
-number = 3.4 # Float | None
-double = 3.4 # Float | None
+number = 8.14 # Float | None
+double = 1.2 # Float | None
 pattern_without_delimiter = 'pattern_without_delimiter_example' # String | None
-byte = 'byte_example' # String | None
+byte = 'BYTE_ARRAY_DATA_HERE' # String | None
 opts = {
   integer: 56, # Integer | None
   int32: 56, # Integer | None
-  int64: 56, # Integer | None
+  int64: 789, # Integer | None
   float: 3.4, # Float | None
   string: 'string_example', # String | None
-  binary: File.new('/path/to/file'), # File | None
+  binary: File.new('/path/to/some/file'), # File | None
   date: Date.parse('2013-10-20'), # Date | None
-  date_time: Time.parse('2013-10-20T19:20:30+01:00'), # Time | None
+  date_time: DateTime.parse('2013-10-20T19:20:30+01:00'), # Time | None
```

Comment:
@zippolyte Shall we use `Time` instead?

Ref: https://github.com/OpenAPITools/openapi-generator/pull/7656